Anima/registration/algorithms/animaKissingSymmetricBMRegistrationMethod.h

```diff
@@ -26,15 +26,25 @@ class KissingSymmetricBMRegistrationMethod : public anima::BaseBMRegistrationMet
 
     itkNewMacro(Self)
 
+    itkSetMacro(ReferenceBackgroundValue, float)
+    itkSetMacro(FloatingBackgroundValue, float)
+
 protected:
-    KissingSymmetricBMRegistrationMethod() {}
+    KissingSymmetricBMRegistrationMethod()
+    {
+        m_ReferenceBackgroundValue = 0;
+        m_FloatingBackgroundValue = 0;
+    }
+
     virtual ~KissingSymmetricBMRegistrationMethod() {}
 
     virtual void PerformOneIteration(InputImageType *refImage, InputImageType *movingImage, TransformPointer &addOn) ITK_OVERRIDE;
 
 private:
-    KissingSymmetricBMRegistrationMethod(const Self&); //purposely not implemented
```

Comment:
I suggest :
    KissingSymmetricBMRegistrationMethod(const Self&) = delete;
    void operator=(const Self&) = delete;
Into public zone

Reply:
ITK_DISALLOW_COPY_AND_ASSIGN does this for you. It's to respect itk coding style

Reply:
Cool